population-pyramid/index.js

```diff
@@ -0,0 +1,206 @@
+import * as d3 from 'd3';
+import * as gAxis from 'g-axis';
+import * as gLegend from 'g-legend';
```

Comment:
`3:13  error  'gLegend' is defined but never used  no-unused-vars`

Please remove if unused.